src/core/include/openvino/pass/manager.hpp

```diff
@@ -23,8 +23,11 @@ class OPENVINO_API Manager {
     Manager();
     virtual ~Manager();
 
+    //// \brief Construct Manager with a provided name.
+    explicit Manager(std::string name);
```

Comment:
why "UnnamedManager" is not used here as well?

Reply:
We want to collect information about managers/transformations execution time in a separate job. We need this name parameter here to identify manager in debug log

Reply:
Added another PR https://github.com/openvinotoolkit/openvino/pull/25842 to fix it